### PR TITLE
[Backport to 5.13] - Adding quantity quota alerts

### PR DESCRIPTION
**Pull request by dannyzaken**
### Explain the changes
1. backported PRs
   1. https://github.com/noobaa/noobaa-operator/pull/1117
   2. https://github.com/noobaa/noobaa-operator/pull/1125
    

### Issues: Fixed #xxx / Gap #xxx
1.  BZ https://github.com/noobaa/noobaa-operator/pull/1117

### Testing Instructions:
1. 

- [ ] Doc added/updated
- [ ] Tests added

### CODE REVIEW

**Review by liranmauda — APPROVED** [review]

File: deploy/internal/prometheus-rules.yaml [hunk 1]
```diff
@@ -136,6 +136,28 @@ spec:
       for: 5m
       labels:
         severity: warning
+    - alert: NooBaaBucketReachingQuantityQuotaState
+      annotations:
+        description: A NooBaa bucket {{ $labels.bucket_name }} is using {{ printf \"%0.0f\" $value }}% of its quantity quota
```
Comment (v-harihar) [inline deploy/internal/prometheus-rules.yaml:141]:
```suggestion
        description: A NooBaa bucket {{ $labels.bucket_name }} is using {{ printf "%0.0f" $value }}% of its quantity quota
```

Comment (v-harihar) [reply deploy/internal/prometheus-rules.yaml:141]: as per https://github.com/noobaa/noobaa-operator/pull/1127/files

File: deploy/internal/prometheus-rules.yaml [hunk 2]
```diff
@@ -136,6 +136,28 @@ spec:
...
+        severity_level: warning
+        storage_type: NooBaa
+      expr: |
+        NooBaa_bucket_quantity_quota{bucket_name=~".*"} > 80
+      for: 5m
+      labels:
+        severity: warning
+    - alert: NooBaaBucketExceedingQuantityQuotaState
+      annotations:
+        description: A NooBaa bucket {{ $labels.bucket_name }} is exceeding its quantity quota - {{ printf \"%0.0f\" $value }}% used
```
Comment (v-harihar) [inline deploy/internal/prometheus-rules.yaml:152]:
```suggestion
        description: A NooBaa bucket {{ $labels.bucket_name }} is exceeding its quantity quota - {{ printf "%0.0f" $value }}% used
```

Comment (v-harihar) [reply deploy/internal/prometheus-rules.yaml:152]: as per https://github.com/noobaa/noobaa-operator/pull/1127/files

**Pull request merged**